src/plugins/stake-plugins/generic/util/tarotUtils.ts

```diff
@@ -38,6 +38,11 @@ export interface TarotContractGroups {
   [TarotPoolTokenType.Collateral]: TarotCollateralContracts
 }
 
+/**
+ * These utils have been copied from tarot.to and minimally changed to work in the stake plugin.
+ * To avoid math errors and interoperability between utils, no code cleanups were attempted.
```

Comment:
Is there source code you can link to (GitHub or such)? That could be helpful to get more information later on if needed.

Reply:
Take the src downloaded from tarot.to and put create a git repo to publish privately on EdgeApp

Reply:
created https://github.com/EdgeApp/www.tarot.to